### DISCUSSION TITLE
(LTSP 19) client no login/exist in client side to login (solved)

### DISCUSSION THREAD

**Discussion by doguibnu** [ltsp/ltsp#164] [Q&A]
Hello

All steps were following about Ltsp 19 and ok nice! Client side UP ok but show only the user from LTSP server (server admin user) for login. If I try insert user manually to login, it tell me that user does not exist and not login. 
Did user were created in Ltsp server?
Yes

What is the Distro?
Linux Mint 19.1 64 bits

How many NICs?
Two! 
1 - Internet (no dhcp)
2  - Ltsp (192.168.67.1) 
All works fine

Please, How can I fix this?
Thanks attention and Help

Douglas 
@alkisg

**Comment by alkisg** [comment]
Hi Douglas,

In LTSP, the users are added on the server.
In the new LTSP, after adding more users, you need to run `ltsp initrd` and reboot the clients.

**Comment by enaut** [comment]
@alkisg does one need to redo the `ltsp initrd` even if the username is entered manually?

**Comment by alkisg** [comment]
@enaut, yes, we use "pam_exec" which requires that user accounts are available locally, even if they're authenticated remotely.

We can probably automate this a bit in the future, either as a "push new users function" of the server or as a "pull new users function" of the clients, but for now, every time a new user is added, the administrator needs to run `ltsp initrd` and reboot the client where the user will log in.

**Comment by doguibnu** [comment]
> Hi Douglas,
> 
> In LTSP, the users are added on the server.
> In the new LTSP, after adding more users, you need to run `ltsp initrd` and reboot the clients.

Solved! Nice!
Thank you so Much
